azure-pipelines.yml

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+pool: 'STL'
+
+steps:
+- task: BatchScript@1
+  inputs:
+    filename: 'azure-devops/enforce-clang-format.cmd'
+    failOnStandardError: true
+    displayName: 'Verify clang-format'

```

Comment:
Enforce, or verify? Terminology should be consistent. ALL SHALL LOVE CLANG-FORMAT AND DESPAIR.